src/confd/yang/infix-system.yang

```diff
@@ -256,6 +301,37 @@ module infix-system {
     }
   }
 
+  augment "/sys:system-state" {
+    container ntp {
+      description "NTP status";
+      container sources {
+        list source {
+          key address;
+          leaf address {
+            type inet:ip-address;
+            description "Address to connected NTP server";
+          }
+          leaf source-mode {
+            type source-mode;
+            description "Source mode";
+          }
+          leaf source-state {
+            type source-state;
+            description "Source state";
+          }
```

Comment:
As I mentioned in passing IRL, do we need the `source-` prefix for these nodes?  Are we expecting a "target-mode" in the source list?  Maybe just `mode` and `state` are sufficient, with a description similar to "Server mode" and "Server state"?